requirements-pinned.txt

```diff
@@ -35,6 +35,6 @@ pathspec==0.8.0
 pycparser==2.20           # via cffi
 pynacl==1.4.0             # via securesystemslib
 python-dateutil==2.8.1
-securesystemslib[crypto,pynacl]==0.16.0
+git+https://github.com/lukpueh/securesystemslib@misc-interface-enhancements
```

Comment:
we should just be sure to update this whenever releasing. I wonder what's the best way to signal this

Reply:
Oh, I probably would have waited until secure-systems-lab/securesystemslib#288 was merged. But I'm sure I won't forget to remove the temporary pinning commit f1f02f4 before we release 1.0.0.